packages/codemod/src/helpers/separateImports.ts

```diff
@@ -0,0 +1,60 @@
+import { Collection, ImportSpecifier, JSCodeshift } from "jscodeshift";
+
+/**
+ * Separate imports from currentLibName to nextLibName
+ * import { Create, Button, Form } from "@pankod/refine-antd";
+ * to
+ * import { Create } from "@pankod/refine-antd";
+ * import { Button, Form } from "antd";
+ */
+export default function separateImports(payload: {
+    j: JSCodeshift;
+    source: Collection<any>;
+    imports: string[];
+    renameImports: { [key: string]: string };
+    currentLibName: string;
+    nextLibName: string;
+}) {
+    const { j, source, imports, renameImports, currentLibName, nextLibName } =
+        payload;
+    const nextLibImports: ImportSpecifier[] = [];
+
+    const refineImport = source.find(j.ImportDeclaration, {
+        source: {
+            value: currentLibName,
+        },
+    });
+
+    refineImport.replaceWith((path) => {
+        for (const item of path.node.specifiers) {
+            if (imports.includes(item.local.name)) {
```

Comment:
```suggestion
			if (imports.includes(item.local.name) || imports.includes(item.imported.name))
```

I believe we should support "as" imports too.
https://astexplorer.net/#/gist/95a3a6e7f8c49482c38066554efb0702/1c5b31e6344cf56ec0c07afabd93979906e4fc11